### PR TITLE
[14.0][MIG] account_bank_statement_import

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Review by alexey-pelykh — APPROVED** [review]

**Review by njeudy — APPROVED** [review]
test and ok for me thanks @alexis-via

File: account_statement_import/i18n/account_bank_statement_import.pot [hunk 1]
```diff
@@ -0,0 +1,800 @@
+# Translation of Odoo Server.
```
Comment (i-vyshnevska) [inline account_statement_import/i18n/account_bank_statement_import.pot]: remove file

File: account_statement_import/wizard/account_statement_import.py [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+                -o 'balance_start': float (e.g: 8368.56)
+                -o 'balance_end_real': float (e.g: 8888.88)
+                - 'transactions': list of dict containing :
+                    - 'payment_ref': string (label of the line)
+                    - 'date': date
+                    - 'amount': float
+                    - 'unique_import_id': string
+                    -o 'account_number': string
+                        Will be used to find/create the res.partner.bank in odoo
+                    -o 'note': string
```
Comment (i-vyshnevska) [inline account_statement_import/wizard/account_statement_import.py]: field "note" was removed from the model

**Review by i-vyshnevska — CHANGES_REQUESTED** [review]
@alexis-via you need to keep the initial commit history

File: account_statement_import/wizard/account_statement_import.py [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+                        "res_id": result["statement_ids"][0],
+                    }
+                )
+            else:
+                action["domain"] = [("id", "in", result["statement_ids"])]
+        else:
+            # dispatch to reconciliation interface
+            line_ids = self.env["account.bank.statement.line"].search(
+                [("statement_id", "in", result["statement_ids"])]
+            )
```
Comment (luc-demeyer) [inline account_statement_import/wizard/account_statement_import.py:56]: search returns recordset -> change to search([...]).ids

**Review by luc-demeyer — APPROVED** [review]

**Comment by luc-demeyer** [comment]
@alexis-via 
note field has not been removed but is renamed to 'narration'

**Comment by pedrobaeza** [comment]
Some git work is needed before merging:

- Change first commit author to be something like info@odoo.com for changing attribution.
- Pre-commit stuff should be before doing any change.
- Squash unneeded changes.

@alexis-via do you want me to do these changes?

**Comment by alexis-via** [comment]
@pedrobaeza Yes, go ahead !

**Comment by fclementic2c** [comment]
Hello, I would appreciate to have this PR merged quickly if possible. Thanks @alexis-via

**Review by fclementic2c — APPROVED** [review]

**Comment by pedrobaeza** [comment]
As I can't push to this branch being a corporate branch (don't know why GitHub applies such constraint), I have opened #367, and I will directly merge it as this one has enough approvals.

**Pull request closed without merging**